configure

```diff
@@ -1045,15 +1045,15 @@ def configure_intl(o):
         if nodedownload.candownload(auto_downloads, "icu"):
           nodedownload.retrievefile(url, targetfile)
       else:
-        print(' Re-using existing %s' % targetfile)
+        print('Re-using existing %s' % targetfile)
       if os.path.isfile(targetfile):
         sys.stdout.write(' Checking file integrity with MD5:\r')
```

Comment:
Ah good point, I only searched for print statements. I'll update the write statements too, thanks.